codex-rs/app-server-protocol/src/protocol/v2.rs

```diff
@@ -527,10 +622,18 @@ pub enum ThreadItem {
     #[ts(rename_all = "camelCase")]
     CommandExecution {
         id: String,
+        /// The command to be executed.
         command: String,
-        aggregated_output: String,
-        exit_code: Option<i32>,
+        /// The command's working directory if not the default cwd for the agent.
```

Comment:
This says "if not" but this is not `Option<PathBuf>`? We should keep `cwd` as required, right?